scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -485,6 +493,9 @@ def create_config_proto(params):
     rewriter_config = rewriter_config_pb2.RewriterConfig()
     text_format.Merge(params.rewriter_config, rewriter_config)
     config.graph_options.rewrite_options.CopyFrom(rewriter_config)
+  if params.variable_update == 'horovod':
+    import horovod.tensorflow as hvd
```

Comment:
Here and everywhere you import horovod, add `  # pylint: disable=g-import-not-at-top` to the end of the line. Make sure there are two spaces between `hvd` and `#`. Don't worry about exceeding the line length for this comments.